subsys/bluetooth/audio/bap_iso.c

```diff
@@ -144,8 +144,35 @@ void bt_bap_iso_init(struct bt_bap_iso *iso, struct bt_iso_chan_ops *ops)
 	iso->chan.qos->tx->path->cc = iso->tx.cc;
 }
 
+static struct bt_bap_iso_dir *bap_iso_get_iso_dir(bool unicast_client, struct bt_bap_iso *iso,
+						  enum bt_audio_dir dir)
+{
+	/* TODO FIX FOR CLIENT */
+	if (IS_ENABLED(CONFIG_BT_BAP_UNICAST_CLIENT) && unicast_client) {
+		/* For the unicast client, the direction and tx/rx is reversed */
+		if (dir == BT_AUDIO_DIR_SOURCE) {
+			return &iso->rx;
+		} else {
+			return &iso->tx;
+		}
+	}
+
+	if (dir == BT_AUDIO_DIR_SINK) {
+		return &iso->rx;
+	} else {
+		return &iso->tx;
+	}
+}
+
+static inline bool is_unicast_client_ep(struct bt_bap_ep *ep)
```

Comment:
```suggestion
static bool is_unicast_client_ep(struct bt_bap_ep *ep)
```
No reason to inline this, is there?